styles/discuit.org/catppuccin.user.less

```
@@ -0,0 +1,197 @@
+/* ==UserStyle==
+@name discuit.org Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/discuit.org
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/discuit.org
+@version 2000.01.01
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/discuit.org/catppuccin.user.less
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Adiscuit.org
+@description Soothing pastel theme for discuit.org
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document regexp("^https?://discuit\\.(org|net)/.*") {
```

Can we simplify this to below or are we explicitly excluding some subdomains or something?
```suggestion
@-moz-document domain("discuit.org"), domain("discuit.net") {
```